ecs-cli/modules/cli/local/down_app.go

```diff
@@ -42,8 +42,30 @@ func Down(c *cli.Context) error {
 		network.Teardown(client)
 	}()
 
+	if err := optionsPreCheck(c); err != nil {
+		logrus.Fatalf("Tasks can be either created by local files or remote files")
+	}
+	// if c.Bool(flags.AllFlag) {
+	// 	return downLocalContainersWithFilters()
+	// }
+	// return downComposeLocalContainers()
+
+	if c.String(flags.TaskDefinitionFileFlag) != "" {
+		return downLocalContainersWithFilters(filters.NewArgs(
+			filters.Arg("label", taskDefinitionLabelValue+"="+c.String(flags.TaskDefinitionFileFlag)),
+			filters.Arg("label", taskDefinitionLabelType+"="+"localFile"),
```

Comment:
Can we move `remoteFile` and `localFile` to constants? Ideally, these should be defined in `create_app.go`. We have a note to fix these duplications here: https://github.com/aws/amazon-ecs-cli/projects/2#card-22758290

Reply:
Oh that is also what I was thinking about. I was about to do that. Probably tomorrow.